ql/src/java/org/apache/hadoop/hive/ql/metadata/HiveUtils.java

```diff
@@ -304,6 +304,10 @@ public static HiveStorageHandler getStorageHandler(
     }
   }
 
+  public static String getDefaultComment(HiveStorageHandler storageHandler) {
+    return storageHandler != null ? storageHandler.getDefaultColumnComment() : HiveStorageHandler.FROM_SERIALIZER;
```

Comment:
Doesn't this add "from deserializer" to native tables (where storage handler is null)? I thought this default comment thing was only related to non-native tables

Reply:
Default storage handler still could have a `from deserializer` field. For example AVRO files can contain the schema and they can dictate the columns of the table. This change makes sure that we do not get an error when the storage handler is null. The other default is handled by the default implementation for the `getDefaultColumnComment()` method